catalog/spatial/registry/registry-rest-endpoint/pom.xml

```diff
@@ -50,11 +50,7 @@
         <dependency>
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
-            <version>3.4</version>
-        </dependency>
-        <dependency>
-            <groupId>org.codice.ddf.registry</groupId>
-            <artifactId>registry-schema-bindings</artifactId>
+            <version>3.0</version>
```

Comment:
I think wrapping it is the best choice. It's easy enough to do and to upgrade later and matches what we've done with Guava.